test/ITensorLegacyMPS/base/test_dmrg.jl

```diff
@@ -110,7 +110,14 @@ using ITensors: nsite, set_nsite!
     @test length(PHdisk) == N
     @test ITensors.site_range(PH) == n:(n + 1)
     @test eltype(PH) == Float64
-    @test size(PH) == (3^2 * 4^2, 3^2 * 4^2)
+    ## TODO sometimes randomMPS gives a linkdim value of 3 
+    ## which causes an error in `size = 3^2 * 4^2`
+    size =
```

Comment:
You should use a variable name other than `size` since that overwrite the function`size(...)`.

Reply:
Make sense, thanks!